be/src/olap/base_compaction.cpp

```diff
@@ -124,7 +124,30 @@ OLAPStatus BaseCompaction::run() {
     // 3. 执行base compaction
     //    执行过程可能会持续比较长时间
     stage_watch.reset();
+    RowsetId rowset_id = 0;
+    RETURN_NOT_OK(_tablet->next_rowset_id(&rowset_id));
+    RowsetWriterContextBuilder context_builder;
+    context_builder.set_rowset_id(rowset_id)
```

Comment:
Build should have default value, otherwise we can create class instantly.

Reply:
ok